src/bin/pg_dump/pg_dump.c

```diff
@@ -2166,6 +2181,7 @@ dumpTableData_insert(Archive *fout, const void *dcontext)
 	if (tdinfo->filtercond)
 		appendPQExpBuffer(q, " %s", tdinfo->filtercond);
 
+	getCursorForBbfCatalogTableData(fout, tbinfo, q, &nfields);
```

Comment:
Rather than `get`, I think this is more like `fix` or `reset` because it resets `PQExpBuffer` and `nfields`. Same arguments can be applied for other function names.

Reply:
Makes sense.